### PR TITLE
Add typography section to styleguide

### PR DESCRIPTION
**Pull request by jimbo**
## Description
Add a typography section to the styleguide. The prose content may be lorem ipsum, but the values should reflect the the official Venia styles.

## Related Issue
PWA-153

## Acceptance 
- @jcalcaben should affirm document structure
- @soumya-ashok should affirm Venia typographic styles
- Others may review the code

### Verification Stakeholders
@jcalcaben @soumya-ashok 

### Verification Steps

1. `yarn workspace @magento/venia-styleguide run start`
2. Navigate to `/page/typography`

## Checklist
* I have updated the documentation accordingly, if necessary.
* I have *not* added tests to cover my changes, if necessary.

### CODE REVIEW

File: packages/venia-styleguide/src/pages/typography/sections/Formatting/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+Semantic text formatting adds importance or emphasis to words or phrases.
+
+***
+
+<Columns reverse={true}>
+  <Strong />
+</Columns>
+<Columns reverse={true}>
+  <Emphasis />
+</Columns>
```
Comment (sirugh) [inline packages/venia-styleguide/src/pages/typography/sections/Formatting/index.md]: This looks weird -- should the text be contained within the grey-box background? 

[Image from Gyazo](https://gyazo.com/5fa4bef7d3d90e5853e72e9c5df1198f)

File: packages/venia-styleguide/src/pages/typography/Typography.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+<Section title="Hierarchy">
+    <Hierarchy />
+</Section>
+<Section title="Font sizes">
+    <FontSizes />
+</Section>
+<Section title="Formatting">
+    <Formatting />
+</Section>
+<Section title="Usage guidelines" />
```
Comment (sirugh) [inline packages/venia-styleguide/src/pages/typography/Typography.md]: This is empty. Drop some lorem text placeholder?

File: packages/venia-styleguide/src/pages/typography/sections/FontSizes/index.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import Columns from "../../../../components/Columns"
+import FontSize from "./FontSize"
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce quis euismod nisi. Morbi metus mauris, volutpat ac aliquet eget, laoreet vel tortor. Pellentesque commodo tellus nibh, vitae varius lectus pharetra in. Aliquam quis nisi ligula. Proin sit amet mauris ac lacus efficitur varius eget in urna. Ut sagittis feugiat ex et dictum. Nam ut tempor urna, at dapibus erat. Aenean ac dui a tellus venenatis accumsan.
+
+***
+
+<Columns>
+  <FontSize />
```
Comment (sirugh) [inline packages/venia-styleguide/src/pages/typography/sections/FontSizes/index.md]: Maybe it's the columns component but it looks like it makes two columns differing by background.

File: packages/venia-styleguide/src/components/Article/Article.css [hunk 4]
```diff
@@ -0,0 +1,4 @@
+.title {
+    font-size: var(--venia-global-text-fontSize-1100);
```
Comment (sirugh) [inline packages/venia-styleguide/src/components/Article/Article.css]: This feels like it should use a typography token for heading but I didn't find one that used `1100`.

File: packages/venia-styleguide/src/components/Heading/index.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+import Heading from './Heading';
+
+export default Heading;
+export const H1 = props => <Heading {...props} level={1} />;
+export const H2 = props => <Heading {...props} level={2} />;
+export const H3 = props => <Heading {...props} level={3} />;
+export const H4 = props => <Heading {...props} level={4} />;
+export const H5 = props => <Heading {...props} level={5} />;
+export const H6 = props => <Heading {...props} level={6} />;
```
Comment (sirugh) [inline packages/venia-styleguide/src/components/Heading/index.js]: These don't seem to be used anywhere. Are these supposed to be components for the styleguide UI or for use in PWA Studio (why not both?)

Comment (sirugh) [reply packages/venia-styleguide/src/components/Heading/index.js]: Oh nvm I see the `MDXAdapter` :)

**Comment by dpatil-magento** [comment]
Verification steps looks good. Need @jcalcaben @soumya-ashok approval to merge this one.

**Pull request merged**